Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -64,14 +64,15 @@ def contributions
         return ofail "Unsupported repository: #{repo}. Try one of #{SUPPORTED_REPOS.join(", ")}."
       end
 
+      tap = Tap.fetch("Homebrew", repo)
       repo_path = find_repo_path_for_repo(repo)
       unless repo_path.exist?
         opoo "Repository #{repo} not yet tapped! Tapping it now..."
-        Tap.fetch("homebrew", repo).install
+        tap.install
       end
 
       results[repo] = {
-        commits:       git_log_author_cmd(T.must(repo_path), args),
+        commits:       GitHub.repo_commit_count_for_user(tap.full_name, args.named.first),
```

Comment:
For backwards compatibility (and accuracy of "First Last" name queries), maybe we should keep `git_log_author_cmd` around until everything is made to use the GitHub APIs and call a different method for commit counts depending on if `args.named.first` is a GitHub username or a full name?

Reply:
I don't think we need backwards compatibility for this command FWIW.